src/scripting/Convo.js

```diff
@@ -62,7 +62,15 @@ class Convo {
           if (convoStep.sender === 'me') {
             convoStep.messageText = this._checkNormalizeText(container, scriptingMemory, convoStep.messageText)
             debug(`${this.header.name}/${convoStep.stepTag}: user says ${JSON.stringify(convoStep, null, 2)}`)
-            container.UserSays(new BotiumMockMessage(convoStep))
+
+            new Promise(resolve => {
+              if (container.caps.SIMULATE_WRITING_SPEED) {
+                setTimeout(() => resolve(), container.caps.SIMULATE_WRITING_SPEED * convoStep.messageText.length)
```

Comment:
messageText could be missing (for click&point bot interfaces ...). please consider this here.

Reply:
done